src/DataStructures/Tensor/Expressions/TensorExpression.hpp

```diff
@@ -25,7 +54,7 @@
  * Used to denote a tensor index in a tensor slot. This allows the following
  * type of expressions to work:
  * \code{.cpp}
- * auto T = evaluate<ti_a_t, ti_b_t>(F(ti_a, ti_b) + S(ti_b, ti_a));
+ * auto T = evaluate<ti_a_t, ti_B_t>(F(ti_a, ti_B) + S(ti_B, ti_a));
```

Comment:
Good eye. I did intentionally change this when I changed `TensorIndex` to be templated on valence. However, I didn't change this back after the rework for the `TensorIndex` encodings with the sentinel, where it makes more sense to show the example with `ti_b` over `ti_B`, because `TensorIndex<1>` is more intuitive than `TensorIndex<500>` for an example. I've changed this back to use `ti_b`. Thanks!